### PR TITLE
[Merged by Bors] - Implement pause/stop

### PR DESCRIPTION
**Pull request by razvan**
- added cluster operations - wip
- added test cases
- clusteroperation defaults now

# Description

*Please add a description here. This will become the commit message of the merge request later.*

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Review by soenkeliebau — APPROVED** [review]
I have looked this over on my mobile, so never ran it, but from what I can see this looks good to me.

File: docs/modules/zookeeper/pages/usage_guide/cluster_operations.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+= Cluster Operation
+
+The managed ZooKeeper instances can be configured with different cluster operations. See xref:concepts:cluster_operations.adoc[] for more details.
```
Comment (fhennig) [inline docs/modules/zookeeper/pages/usage_guide/cluster_operations.adoc]:
```suggestion
The managed ZooKeeper instances can be configured with different xref:concepts:cluster_operations.adoc[cluster operations].
```

**Comment by maltesander** [comment]
test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/zookeeper-operator-it-custom/39/

**Review by maltesander — APPROVED** [review]
bors merge

**Pull request closed without merging**